src/main/java/com/javachallengers/simpson/service/SimpsonCharacterService.java

```diff
@@ -0,0 +1,32 @@
+package com.javachallengers.simpson.service;
+
+import java.util.Optional;
+
+import com.javachallengers.simpson.exception.SimpsonCharacterException;
+import com.javachallengers.simpson.model.SimpsonCharacter;
+import com.javachallengers.simpson.model.dto.SimpsonCharacterRequestDTO;
+
+/**
+ * Service with business rules related to Simpson Characters
+ */
+public interface SimpsonCharacterService {
```

Comment:
Do you think there is benefit of using the SimpsonCharacterService since we won't be using polymorphism here?

Reply:
I don't see any use of polymorphism but I tried to make the class depending on abstractions over implementations to follow the dependency injection and SOLID principles. What would be the best practice about it?

Reply:
I think we can have just the class implementation. We don't need to follow SOLID principles very strongly. 

I say that because I've been working on a project with around 70 developers with many and many ServiceInterfaces and those interfaces were never used in the project. So we concluded we were creating extra complexity for no gain.

Reply:
I agree with you. I've faced a similar problem in my current project.

Reply:
Removed the interface and using the implementation directly.